File: encrypt_decrypt.py
#pip install pycryptodome
from Crypto.Cipher import AES


def encrypt(key, plaintext):
    plaintext_b = plaintext.encode('ascii')

    cipher = AES.new(key, AES.MODE_EAX)
    ciphertext, tag = cipher.encrypt_and_digest(plaintext_b)

    return ciphertext,cipher.nonce


def decrypt(key, ciphertext,nonce):
    cipher = AES.new(key, AES.MODE_EAX,nonce=nonce)
    plaintext = cipher.decrypt(ciphertext)

    return plaintext.decode('ascii')

File: encrypt_decrypt.ts
import * as AES from 'aes';

export function encrypt(key,plaintext){
  let aes_instance = new AES(key);
  return aes_instance.encrypt(plaintext)
}

export function decyrpt(key,ciphertext){
  let aes_instance = new AES(key);
  return aes_instance.decyrpt(ciphertext)
}
